app/controllers/conferences_controller.rb

```diff
@@ -33,8 +32,15 @@ def redirect
 
   private
 
-  def generate_gid(team)
-    "#{Season.current.name}-#{Time.now.getutc.to_i}-#{team.id}"
+  def build_conference
+    conference = Conference.new(conference_params)
+    conference.season_id = current_season.id
```

Comment:
You can set the season_id at the same time as the conference_params like this:
`conference = Conference.new(conference_params.merge(season: current_season))`
The same goes (I think) for the generated `gid`, but I don't mind if you leave that separately.